cedar-testing/src/cedar_test_impl.rs

```diff
@@ -203,6 +205,18 @@ pub trait CedarTestImplementation {
         mode: ValidationMode,
     ) -> TestResult<TestValidationResult>;
 
+    fn validate_request(
+        &self,
+        schema: &ValidatorSchema,
+        request: &Request,
+    ) -> TestResult<TestValidationResult>;
+
+    fn validate_entities(
+        &self,
+        schema: &ValidatorSchema,
+        entities: Entities,
```

Comment:
Probably want to take a reference here (`&Entities`) since that matches what we do elsewhere.